app/tests/test_service_site.py

```diff
@@ -0,0 +1,86 @@
+from unittest.case import skip
+from starlette import responses
+from ..main import app
+from fastapi.testclient import TestClient
+import bson 
+import unittest
+import requests_mock
+
+class TestServiceSite(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.client = TestClient(app)
+        self.base_url = "/api"
+    
+    def test_get_all_service_site(self):
+        """Test retrive all the valid service sites."""
+        responses = self.client.get(f"{self.base_url}/sites")
+        self.assertEqual(200, responses.status_code)
+
+    def test_get_service_site(self):
+        """Test retrive specific service sites using its id which is 24 characters."""
+        responses = self.client.get(
+            f"{self.base_url}/site/617923857ad4eeefea76d121")
+        self.assertEqual(200, responses.status_code)
```

Comment:
```
response -> name
  field required (type=value_error.missing)
response -> location
  field required (type=value_error.missing)
```
These fields are required, making the tests fail

Reply:
resolve on [dda79a9](https://github.com/OGYH-Team/ogyh-backend/pull/6/commits/dda79a98829716ae7ead4f433b1e074c0e6d0459)